pkg/kv/kvserver/allocator/storepool/store_pool.go

```diff
@@ -41,6 +42,10 @@ const (
 	// TestTimeUntilStoreDeadOff is the test value for TimeUntilStoreDead that
 	// prevents the store pool from marking stores as dead.
 	TestTimeUntilStoreDeadOff = 24 * time.Hour
+
+	// IOOverloadTrackedRetention is the period over which to accumulate
```

Comment:
I'm removing the windowing all together - see above reply. I've expanded the comment and made the value a retention period, rather than a width for sliding window buckets.